cypress/integration/Manual Listing Pages/ManualListingPages.feature

```diff
@@ -0,0 +1,157 @@
+Feature: All manual pages
+
+    Scenario Outline: All manual trial listing page components are displayed(with pagers)
+        Given screen breakpoint is set to "<screen breakpoint>"
+        When user is navigating to "<path>"
+        Then page title is "<page title>"
+        And the page displays pager info "Trials 1-N of N"
+        And the 1 pager is displayed with the following "<pager items>"
+        And the page "1" is highlighted
+        And each result displays the trial title with a link in the following format "/clinicaltrials/NCI-"
+        And each result displays the trial summary
+        And each result displays "Location: " below the summary
+        And the 2 pager is displayed with the following "<pager items>"
+
```

Comment:
Please add last step here :
'And delighter is displayed'
Implement the step - it will fail for now for 11 pages ( only 2 have it on ode), but it will be passing once hot fix makes it out.